lib/index.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Modal from './index';
+
+describe('Modal', () => {
+    it('renders', () => {
+        expect(shallow((
+            <Modal isOpen={false}>
+                Test
+            </Modal>
+        ))).toMatchSnapshot();
+    });
+
+    it('#isOpen', () => {
+        expect(shallow((
+            <Modal isOpen={true}>
+                Test
+            </Modal>
+        ))).toMatchSnapshot();
+    });
+});
```

Comment:
Very poor coverage. (I know it's not the purpose of this PR though)
I will see if I can improve this :)